src/flare/flr-dca-cooldown.rain

```diff
@@ -0,0 +1,76 @@
+tokens:
+  eusdt:
+    address: 0x96B41289D90444B8adD57e6F265DB5aE8651DF29
+    decimals: 6
+  wflr:
+    address: 0x1D80c49BbBCd1C0911346656B529DF9E5c2F783d
+    decimals: 18
+
+orders:
+  dca-buy:
+    inputs:
+        # EUSDT
+        - token: 0x96B41289D90444B8adD57e6F265DB5aE8651DF29
```

Comment:
this is off spec isn't it? don't we directly reference defined tokens by name here?